src/app/pages/ledger/components/connect-ledger-title.tsx

```diff
@@ -0,0 +1,9 @@
+import { Title } from '@app/components/typography';
+
+export function ConnectLedgerTitle() {
+  return (
+    <Title fontSize={3} lineHeight="1.4" mt="base" width="290px">
```

Comment:
Again, I have no context of other parts of the codebase so take this comment for what it is:

why does the title have a fixed width? By being fixed, this UI component is not flexible to accommodate different viewport sizes if we ever want to make this component responsive.

In general, any fixed `width` should be avoided, a better option here would be to add inline margins.

Reply:
I can change it to use margins. Our full page view isn't currently responsive.

Reply:
yeap, I understand that it isn't responsive, and please feel free to ignore it. I am just saying that it's always a good practice to avoid defining `width` since, to make things flexible and composable, a component should not know how wide it is, instead the context in which they are inserted should define it.

Reply:
Makes sense, tbh I debated about which was better, thanks for the feedback. I'll switch to using margins.